command/pr_create.go

```diff
@@ -107,16 +107,6 @@ func prCreate(cmd *cobra.Command, _ []string) error {
 		break
 	}
 
-	isWeb, err := cmd.Flags().GetBool("web")
```

Comment:
This part is moved below, because if its here this can open the web with a different URL. i.e no base branch, which will default to `master` as base branch.